packages/cli/test/integration/workflows.controller.ee.test.ts

```diff
@@ -0,0 +1,117 @@
+import express from 'express';
```

Comment:
This file needs formatting. Please check that you have `"editor.defaultFormatter": "esbenp.prettier-vscode"` and `"editor.formatOnSave": true`

Reply:
Thanks, somehow I was missing `formatOnSave` -_-